docs/semgrep-code/findings.md

```diff
@@ -35,13 +35,19 @@ To access the [**Findings** page](https://semgrep.dev/orgs/-/findings):
 The **Findings** page consists of:
 
 - The **filter panel**, which you can use to group and filter for specific findings
-- Information about findings identified by Semgrep Code. Each  finding in the list includes:
+- Information about findings identified by Semgrep Code. Each finding in the list includes:
   - The name and description of the rule used to generate the finding
   - The name of the project, as well as links to the Git branch and source code, where Semgrep Code identified the finding
 
 ### Group findings
 
-By default, Semgrep groups all of the findings by the rule Semgrep used to match the code. This view is called the **Group by rule** view. The rules that reported the highest number of findings are at the top of the page. This helps you see which rules have the highest efficiency and which generate a lot of noise in your results.
+By default, Semgrep groups all of the findings by the rule Semgrep used to match the code. This view is called the **Group by rule** view.
+
+Semgrep sorts findings by severity. For a given severity, Semgrep further sorts findings as follows:
+
+1. Findings generated by custom rules
+2. Findings generated by [Pro rules](/semgrep-code/pro-rules)
+3. All remaining findings
```

Comment:
technically after the above two categories, we sort by descending issue counts and ascending findings ID. but i dont think they actually need to know this